src/types/adapter.ts

```diff
@@ -267,14 +267,14 @@ export interface ProtocolTokenApy extends Erc20Metadata {
   /**
    * Current apy of protocol pool
    */
-  apyDecimal: string
+  apyDecimal: number
```

Comment:
What do we think about this change?

Having it as a string would make it tricky to work with it afterwards.

Reply:
No strong opinion. Dont see an issue atm